pkg/estimator/pb/types.go

```diff
@@ -40,7 +40,7 @@ type ReplicaRequirements struct {
 	NodeClaim *NodeClaim `json:"nodeClaim,omitempty" protobuf:"bytes,1,opt,name=nodeClaim"`
 	// ResourceRequest represents the resources required by each replica.
 	// +optional
-	ResourceRequest corev1.ResourceList `json:"resourceRequest,omitempty" protobuf:"bytes,2,rep,name=resourceRequest,casttype=k8s.io/api/core/v1.ResourceList,castkey=k8s.io/api/core/v1.ResourceName"`
+	ResourceRequest corev1.ResourceList `json:"resourceRequest,omitempty" protobuf:"bytes,2,rep,name=resourceRequest,casttype=k8s.io/api/core/v1.ResourceList,castkey=k8s.io/api/core/v1.ResourceName,castvalue=k8s.io/apimachinery/pkg/api/resource.Quantity"`
```

Comment:
Can you remind me why it should be updated?

Reply:
I don’t know if this is a historical issue. At least now I have an error(that is the package `k8s.io/apimachinery/pkg/api/resource` won't be auto-imported in the file `generated.pb.go`) when using tools to generate these files. Adding this tag can solve it.

Reply:
Do you know how to reproduce the error?

Reply:
emm, so far it seems like this.

Reply:
really? could you provide me which CI pipeline to run this script on? I'll figure it out.

Reply:
This one: https://github.com/karmada-io/karmada/blob/c2989cc573739fb26b7d76312572b46153d09569/.github/workflows/ci.yml#L58

Reply:
You can move the cloned Karmada repo to `$GOPATH/src/github.com/karmada-io` then have a try. That's exactly how CI works.
1. Clone Karmda to `/home/runner/work/karmada/karmada/src/github.com/karmada-io`, see [this log](https://github.com/karmada-io/karmada/actions/runs/7083180355/job/19275130824?pr=4353#step:2:3).
2. Set GOPTAH with `/home/runner/work/karmada/karmada`, see [this log](https://github.com/karmada-io/karmada/actions/runs/7083180355/job/19275130824?pr=4353#step:3:8).

Reply:
okay, thanks for your remind.